openfl/component/aggregator/aggregator.py

```diff
@@ -82,6 +84,8 @@ def __init__(
         log_memory_usage=False,
         write_logs=False,
         callbacks: Optional[List] = None,
+        persist_checkpoint=True,
+        persistent_db_path=None
```

Comment:
If `persistent_db_path` is None by default, then I suggest setting `persist_checkpoint=False` by default accordingly.

PS: As part of the bump into OpenFL-Security, you should also set the `persist_checkpoint` param in the defaults there:
https://github.com/intel-innersource/frameworks.ai.openfl.openfl-security/blob/develop/client/openfl_security_workspace/workspace/plan/defaults/aggregator.yaml

Reply:
the `None` db path is to use the current workdir i.e. workspace, not to hint that the feature is disabled,
I added it as preparation for the secure-fl where we want to set it to a dedicated path for the encrypted fs.